instagram-app/src/components/PostContainer/CommentSection/Likes/Likes.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import open_heart from './open_heart.png';
```

Comment:
I like your named image imports - keeps your `src={imageSrc}` clear. Like variables in LESS!